ui/components/Metadata.tsx

```diff
@@ -16,17 +16,11 @@ type Props = {
 const Label = styled(Text)`
   padding: ${(props) => props.theme.spacing.xs}
     ${(props) => props.theme.spacing.small};
-  margin-right: ${(props) => props.theme.spacing.xxs};
   border-radius: 15px;
   white-space: nowrap;
   background-color: ${(props) => props.theme.colors.neutralGray};
 `;
 
-const LabelFlex = styled(Flex)`
-  padding: ${(props) => props.theme.spacing.xxs} 0;
-  overflow-x: scroll;
```

Comment:
Does the component still scroll on overflow after removing this?

Reply:
I use wrap with Flex so whenever it's overflowing it'll be displayed in multiple lines